### PR TITLE
Updates to the prettyRainfallPlot

### PR DESCRIPTION
**Pull request by Kdreval**
I have addressed the TODO's in `prettyRainfallPlot` and enhanced functionality to support both genome build projections, optionally take existing MAF data frame or path to a maf file. Updated the function documentation and examples.

### CODE REVIEW

File: R/utilities.R [hunk 1]
```diff
@@ -2553,11 +2570,11 @@ massage_matrix_for_clustering = function(incoming_data,
   # did user provide any features they would like to drop from matrix?
   if(!missing(drop_these_features)){
     message("You provided features to be dropped from matrix, removing them ...")
-    output_data =
```
Comment (mattssca) [inline R/utilities.R:2556]: Is there a way to automatically remove spaces, I see you did quite a few edits of that kind in this script?

File: R/viz.R [hunk 2]
```diff
@@ -1,50 +1,158 @@
...
 #' @return a ggplot2 plot. Print it using print() or save it using ggsave()
 #' @export
-#' @import ggplot2 dplyr cowplot
+#' @import ggplot2 dplyr ggrepel
 #'
 #' @examples
-prettyRainfallPlot = function(this_sample_id,label_ashm_genes=TRUE,chrom,projection="grch37",chromosome){
-  if(missing(this_sample_id)){
-    message("You must provide a sample_id to plot")
+#' prettyRainfallPlot("Raji")
```
Comment (mattssca) [inline R/viz.R:18]: (What's "Raji"?)

**Review by mattssca — APPROVED** [review]
Nice plotting function! Thanks,

File: R/utilities.R (hunk 1 above)
Comment (Kdreval) [reply R/utilities.R:2556]: There is! Very handy! I just started doing this. Here is the Slack thread where we discussed it: https://morinlabsfu.slack.com/archives/C0224H120CU/p1648871548155479?thread_ts=1648836087.433839&cid=C0224H120CU

File: R/viz.R (hunk 2 above)
Comment (Kdreval) [reply R/viz.R:18]: It is a sample id of one of the cell lines

**Review by Kdreval — COMMENTED** [review]
Thanks for review Adam! I am adding some more functionality to this function, will update once done!

File: R/viz.R (hunk 2 above)
Comment (mattssca) [reply R/viz.R:18]: Right, I did assume so.

File: R/utilities.R (hunk 1 above)
Comment (mattssca) [reply R/utilities.R:2556]: Yes, I remember that discussion. Neat!

**Review by mattssca — APPROVED** [review]
Awesome, thanks Kostia!

**Comment by Kdreval** [comment]
Happy to report that the prettyForestPlot now supports SV labeling if the user wants to do so!
This PR is ready for re-review.

**Pull request merged**